keyboards/ploopyco/mouse/keymaps/uqs/keymap.c

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2022 Ulrich Spörlein
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+    DBL_CLK_NO = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case DBL_CLK_NO:
+            if (record->event.pressed) {
+                //SEND_STRING(SS_TAP(X_BTN2) SS_DELAY(300) SS_TAP(X_BTN2) SS_DELAY(300) SS_TAP(X_N));
+                tap_code(KC_BTN2);
```

Comment:
Just a heads up, you may want to look at the deferred executor stuff.  This allows for adding waits, but without hard locks, like this. 

It's a bit more complicated, but it is very rewarding.

Reply:
Thanks, I'll add a TODO, but i'm currently not really using that mouse and it somehow didn't work in-game anyway (maybe too quick still?)